index.d.ts

```diff
@@ -0,0 +1,280 @@
+/// <reference types="node" />
```

Comment:
The node reference doesn't cause issues with only being installed as a dev dependency? Is there something special about it?

Reply:
It's left there with the assumption that people already using TypeScript will already have `@types/node` installed them selfs. It will only be an issue if you run `tsc` and don't have `@types/node` installed. And you only do that if you're compiling TypeScript code.

Reply:
See this conversation of details: https://github.com/elastic/apm-agent-nodejs/pull/926#discussion_r267279580